### PR TITLE
Add support for AIOSP to search only the current document or current project.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
This support was just added to core platform.  So this was just about threading this throuhg as we already supported thsi in NavTo.

### CODE REVIEW

File: src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.cs [hunk 1]
```diff
@@ -6,121 +6,121 @@
...
-using Microsoft.CodeAnalysis.ErrorReporting;
 using Microsoft.CodeAnalysis.NavigateTo;
 using Microsoft.CodeAnalysis.Shared.Extensions;
 using Microsoft.CodeAnalysis.Shared.TestHooks;
 using Microsoft.VisualStudio.Language.NavigateTo.Interfaces;
 using Microsoft.VisualStudio.Utilities;
 using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.Editor.Implementation.NavigateTo
+namespace Microsoft.CodeAnalysis.Editor.Implementation.NavigateTo;
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.cs:17]: view with whitespace off.

File: eng/Directory.Packages.props [hunk 2]
```diff
@@ -74,7 +74,7 @@
     <PackageVersion Include="Microsoft.VisualStudio.LiveShare.WebEditors" Version="3.0.8" />
     <PackageVersion Include="Microsoft.VisualStudio.InteractiveWindow" Version="4.0.0" />
     <PackageVersion Include="Microsoft.VisualStudio.VsInteractiveWindow" Version="4.0.0" />
-    <PackageVersion Include="Microsoft.VisualStudio.Search" Version="17.8.35898-preview.2" />
+    <PackageVersion Include="Microsoft.VisualStudio.Search" Version="17.10.961-preview.2" />
```
Comment (CyrusNajmabadi) [inline eng/Directory.Packages.props:77]:
```suggestion
    <PackageVersion Include="Microsoft.VisualStudio.Search" Version="17.8.35898-preview.2" />
```

**Review by genlu — APPROVED** [review]
:shipit:

**Pull request merged**